tests/beta/test_default_tensor_numba.py

```diff
@@ -0,0 +1,2349 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
```

Comment:
Let's rename this file to `test_default_qubit_numba.py` to refer to the device name.